alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
```

Comment:
I'm not insisting, just mentioning: We added SSL between ELB and EC2 in the past. So stripping this would be a step back. We do not have to trust the AWS network.